core/tests/protractor_desktop/emailDashboard.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for email dashboard page.
+ */
+
+var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
+const { browser } = require('protractor');
+var action = require('../protractor_utils/action.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+
+describe('Email Dashboard', function() {
+  EMAIL_DASHBOARD_URL = '/emaildashboard';
+  var preferencesPage = null;
+
+  beforeAll(async function() {
+    preferencesPage = new PreferencesPage.PreferencesPage();
+    await users.createUser('userA@test.com', 'userA');
+    await users.login('userA@test.com');
+    await preferencesPage.get();
+    await waitFor.pageToFullyLoad();
+    await preferencesPage.toggleEmailUpdatesCheckbox();
+    await users.createAndLoginAdminUser(
+      'management@emaildashboard.com', 'management');
+    await browser.get(EMAIL_DASHBOARD_URL);
+    await waitFor.pageToFullyLoad();
+  });
+
+  it('should query for users', async function() {
+    var schemaEditorElement = element(
+      by.css('.protractor-test-email-dashboard-input-3'));
+    var inputElement = schemaEditorElement.element(by.tagName('input'));
+    var submitButton = element(by.css('.protractor-test-submit-query-button'));
+
+    await action.clear('Email dashboard input 3', inputElement);
+    await action.sendKeys('Email dashboard input 3', inputElement, 1);
```

Comment:
The `keys` argument should be a string to stay consistent with the rest of the code

Reply:
Done.